Sources/TSCBasic/Process.swift

```diff
@@ -113,6 +113,9 @@ public struct ProcessResult: CustomStringConvertible {
     }
 }
 
+/// Alias to make the type's name closer to its purpose.
+public typealias ProcessStdinStream = OutputByteStream
```

Comment:
I'm not sure whether the alias makes things more clear or less clear here.  I understand the intent, but for clients who are already familiar with `OutputByteStream`, the introduction of what seems like a unique type is a bit confusing as well.  I'm not strictly opposed, just mildly concerned about the understandability.  I don't think that calling it an output byte stream is wrong, since it's an output from the perspective of the parent process.

Maybe in a future API overhaul we should actually call them `ReadableByteStream` and `WritableByteStream` or something like that.

Reply:
yeah, I think WritableByteStream might be better... we could do that in a later PR, and have OutputByteStream as a type alias to Writable. I prefer that approach to renaming the type as I'm doing here actually, so I'll revert to OutputByteStream

Reply:
renamed ProcessStdinStream to WritableByteStream for now, with a note to rename OutputByteStream

Reply:
Makes sense.  Until we rename `OutputByteStream` to `WritableByteStream` more globally, do you think it's still worth having a custom alias here or could we leave it as `OutputByteStream` in this API (without an alias)?  I think in QuickHelp it will show up as `WritableByteStream` with at first glance still looks like something special.  So I guess I'm suggesting in the long run renaming `OutputByteStream` to `WritableByteStream` everywhere, but for now keeping it as `OutputByteStream` without an alias.

Reply:
sg, using OutputByteStream